### PR TITLE
Fix/camke install error

### PR DESCRIPTION
**Pull request by fangzhou1227**
# PR Details
## Description

Rewrite carma_client_interface;
Fixes issues found in the integration testing.
## Motivation and Context

Fixes issues found in the integration testing.
## How Has This Been Tested?
Tested in the carla-carma simulation.

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [x] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: carla_twist_to_control/src/carla_twist_to_control/carla_twist_to_control.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# This work is licensed under the terms of the MIT license.
```
Comment (kjrush) [inline carla_twist_to_control/src/carla_twist_to_control/carla_twist_to_control.py:5]: If we don't already, let's make sure we have a `LICENSE` file with the contents of the MIT license in the directory for this package, and probably call out that this 'carla_twist_to_control` package is MIT licensed somewhere in the readme. Might not hurt to check on the licenses of whatever other packages we have here.

**Review by kjrush — CHANGES_REQUESTED** [review]
Looks pretty good to me, let's just double check to make sure we have the carma licensing statements on all the CARMA authored files, and appropriate licensing info for any packages that we've got added here.

File: carla_carma/src/carla_carma/CARMA_client_interface.py [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+
+        Registers on shutdown callback at rospy and spins ROS.
+
+        :return:
+        """
+
+        # init
+        self.sim_time_pub.publish(Clock(self.ros_timestamp))
+        try:
+            self.s.connect(('127.0.0.1', 8080))
```
[the remark this answers is not in the document]
Comment (fangzhou1227) [reply carla_carma/src/carla_carma/CARMA_client_interface.py:268]: This is the IP address for the socket communication between CARLA client (CARLA_server_interface) and CARMA ros nodes. I think making it as augments maybe better. Will update this.

File: carla_twist_to_control/src/carla_twist_to_control/carla_twist_to_control.py (hunk 1 above)
Comment (fangzhou1227) [reply carla_twist_to_control/src/carla_twist_to_control/carla_twist_to_control.py:5]: We are not using this anymore. It was here to replace the ackermann control due to the performance of ackermann control. Do we want to keep it or delete it? If we want to keep it, I will check the license.

File: carla_carma/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 2.8.3)
```
[the remark this answers is not in the document]
Comment (fangzhou1227) [reply carla_carma/CMakeLists.txt:15]: Done

File: carla_carma/launch/carma_comm.launch [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<launch>
```
[the remark this answers is not in the document]
Comment (fangzhou1227) [reply carla_carma/launch/carma_comm.launch:13]: Done

File: carla_carma/src/carla_carma/CARMA_client_interface.py [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+# import carma_srvs
+from std_msgs.msg import String, Header, Bool
+from geometry_msgs.msg import Pose, PoseStamped, Point, Quaternion, PoseWithCovariance, Twist, TwistWithCovariance, Vector3, TwistStamped
+from math import cos, sin, tan
+from rosgraph_msgs.msg import Clock
+
+# When using CARMA_client_interface_updated.py, run CARMA first on the same server.
+# If the interface node and CARMA are on separate computers, set ROS_MASTER_URI variable before rosrun (http://wiki.ros.org/ROS/Tutorials/MultipleMachines).
+
+# TODO: change imports below for where the messages actually are
```
[the remark this answers is not in the document]
Comment (fangzhou1227) [reply carla_carma/src/carla_carma/CARMA_client_interface.py]: Deleted

File: carla_carma/src/carla_carma/CARMA_client_interface.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (fangzhou1227) [reply carla_carma/src/carla_carma/CARMA_client_interface.py]: Update and make socket ip and port as arguments

File: carla_twist_to_control/src/carla_twist_to_control/carla_twist_to_control.py (hunk 1 above)
Comment (fangzhou1227) [reply carla_twist_to_control/src/carla_twist_to_control/carla_twist_to_control.py]: Deleted it

**Review by kjrush — APPROVED** [review]

**Pull request merged**